workshops/views.py

```diff
@@ -407,7 +407,7 @@ class PersonUpdate(LoginRequiredMixin, UpdateViewContext):
     # don't display the 'password' field, reorder fields
     fields = ['personal', 'middle', 'family', 'username', 'may_contact',
               'email', 'gender', 'airport', 'github', 'twitter', 'url',
-              'is_superuser', 'user_permissions']
```

Comment:
No longer showing `user_permissions` ?  I'm +1 on that in this PR (though it's separate from adding notes), but the comment at the top about not displaying `password` should be updated, and `reorder fields` can probably be removed. Otherwise +1 to merge.